sanitizers/detectors/sql_injection.go

```diff
@@ -18,31 +20,33 @@ var syntaxErrors = []*regexp.Regexp{
 	regexp.MustCompile(`\S+ ERROR: syntax error at or near .* \(SQLSTATE 42601\)`),         // PostgreSQL error message
 }
 
-type SQLInjection struct {
-	id    int    // numeric identifier to distinguish between the detectors for the various call sites
-	query string // the SQL query that has been executed
-	err   error  // the error resulting from executing the SQL query
+var _ Detector = (*sqlInjection)(nil)
+
+type sqlInjection struct {
+	id    int
+	cmd   string
```

Comment:
Same as above regarding the comments. Could you also comment on the reason of changing the field name from `query` to `parameter`?

Reply:
see earlier